modules/yieldlabBidAdapter.js

```diff
@@ -1,7 +1,7 @@
 import * as utils from '../src/utils.js'
 import { registerBidder } from '../src/adapters/bidderFactory.js'
 import find from 'core-js-pure/features/array/find.js'
-import { VIDEO, BANNER } from '../src/mediaTypes.js'
+import { VIDEO, BANNER, NATIVE } from '../src/mediaTypes.js'
```

Comment:
Beside this I will need to create another small PR in https://github.com/prebid/prebid.github.io/blob/1174621dbc456ba79a44d4d2de4055fcab547f33/dev-docs/bidders/yieldlab.md#L6

`media_types: video, banner, native`

Reply:
I think we should have a fork of the documentation repo as well and create pull requests in parallel in both forks. This would align best with our current workflow.